ext-common/Ext/ElmFormat.hs

```diff
@@ -20,31 +21,35 @@ import qualified ElmFormat.Cli
 -- import qualified ElmFormat.Render.Text as Render
 import ElmVersion
 import ElmFormat.Messages
+import Reporting.Annotation (Located(..), Region(..), Position(..))
+import CommandLine.InfoFormatter (ToConsole(..))
 
 
-formatWithEmbedded :: Text -> Either ElmFormat.Messages.InfoMessage Text
-formatWithEmbedded inputText = do
-  ElmFormat.Cli.format ElmVersion.Elm_0_19 ("stdin:nofilepath", inputText)
+formatWithEmbedded :: FilePath -> Text -> Either ElmFormat.Messages.InfoMessage Text
+formatWithEmbedded filePath inputText = do
+  ElmFormat.Cli.format ElmVersion.Elm_0_19 (filePath, inputText)
 
 
-format :: Text -> (Either Text Text)
-format text = do
-  case formatWithEmbedded text of
-    Left err ->
-      Left $ Lamdera.show_ err
-    Right formatted ->
-      Right formatted
+format :: FilePath -> Text -> (Either Text Text)
+format filePath text = do
+  case formatWithEmbedded filePath text of
+    Left err -> Left $ toConsole err
+    Right formatted -> Right formatted
 
 
 formatOrPassthrough :: Text -> Text
 formatOrPassthrough text = do
-  case format text of
+  case format "stdin" text of
```

Comment:
Should this be `"stdin:nofilepath"` to match the previous mode?

https://github.com/lamdera/compiler/pull/39/files#diff-85c281d479ab2d11635d5dd622b188cf2c25b0c3142043303a7c065c892bfef3L27